tensorflow/lite/arena_planner.cc

```diff
@@ -237,9 +237,14 @@ TfLiteStatus ArenaPlanner::CalculateAllocations(int first_node, int last_node) {
     }
   }
 
+  // For the case if the graph is empty the node index can be negative since we
+  // substract from the active node, so the node_index can be zero for those
+  // cases
+  if (active_node > 0) {
   // Don't forget to deallocate temporaries of last node.
   TF_LITE_ENSURE_STATUS(
```

Comment:
Indent here.

Reply:
Sure, same is updated in the code.

Regards
Amit